vars/dockerBinaryBuild.groovy

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/groovy
+
+def call (name, dockerhubOrg="feedhenry", dockerHubRepo=name, credentialId="dockerhubjenkins", body) {
+  withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: credentialId, usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD']]) {
+    createOpenshiftDockerSecret "docker.io/${dockerhubOrg}", "${env.USERNAME}", "${env.PASSWORD}", "fh.team.eng-group@redhat.com","${dockerhubOrg}"
+  }
+
+  def pkgVersion = sh(returnStdout: true, script: "node -p -e \"require('./package.json').version\"").trim().split('-')[0]
```

Comment:
This is probably fine, I just wonder if it would be cleaner to parse the package.json in Groovy, rather than spawning a shell so that we can spawn a node process so that we can parse a JSON file? It could maybe even be a util function in the Utils class. What do you think?